src/App.js

```diff
@@ -45,6 +46,7 @@ function App() {
         <>
           <GlobalStyles />
           <Router>
+            <ScrollToTop />
```

Comment:
It's kind of unfortunate that this is a component in my eyes, purely because I generally expect components to show up in the UI, but this can probably stay

Reply:
Yeah :/ Agreed, but otherwise it has to sit in a hook somewhere, which would probably get messier. Apparently this is the recommended solution from React-router